src/components/singleArticle.jsx

```diff
@@ -1,35 +1,55 @@
 import { useParams, Link } from "react-router-dom";
 import { useState, useEffect } from "react";
-import { getSingleArticle } from "../utils/api";
+import { getSingleArticle, updateVotes } from "../utils/api";
 
 
 const SingleArticle = () => {
 
     const [currentArticle, setCurrentArticle] = useState({})
+    const [articleVotes, setArticleVotes] = useState(0)
     const [isLoading, setIsLoading] = useState(true)
+    const [isError, setIsError] = useState(false)
     let {article_id} = useParams()
 
     useEffect(() => {
         getSingleArticle(article_id)
         .then((data) => {
             setCurrentArticle(data.article);
+            setArticleVotes(data.article.votes)
             setIsLoading(false);
         })
-    })
+    },[])
```

Comment:
As Verity hinted at, 'article_id' is missing as a dependency in this useEffect, make sure you address your React warnings. It doesn't harm your code to include it.